src/tasks/solvers.ts

```diff
@@ -82,38 +100,31 @@ const setupSolversTask: () => void = () => {
   subtask(
     "solvers-add",
     "Adds a solver to the list of allowed solvers in GPv2.",
-  ).setAction(async ({ args }, hardhatRuntime) => {
-    if (!args || args.length !== 1) {
-      throw new Error(
-        "Invalid number of arguments. Expected the address of the solver to be added.",
-      );
-    }
-    await addSolver(args[0], hardhatRuntime);
-  });
+  )
+    .addPositionalParam<string>("solver", "The solver account to add.")
+    .addFlag("printTransaction", "Prints the transaction to standard out.")
+    .setAction(async (args, hardhatRuntime) => {
+      await addSolver(args, hardhatRuntime);
+    });
```

Comment:
I believe you can now do this:

```suggestion
    .setAction(addSolver);
```

Same for most other subtasks.